cfgov/jinja2/v1/_includes/macros/filters.html

```diff
@@ -361,6 +364,23 @@
 
 {# A helper macro for filters()
    ========================================================================== #}
+{% macro _errors() %}
+    {% set messages = get_messages(request) %}
+    {%- if messages-%}
+        {% import 'macros/notification.html' as notification %}
```

Comment:
Referencing https://github.com/cfpb/cfgov-refresh/pull/1202 as this will need to be updated here or there (whichever goes in first).